marklogic-data-hub-central/ui/src/components/modal-status/modal-status.test.tsx

```diff
@@ -1,12 +1,211 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render, wait } from '@testing-library/react';
+import axiosMock from 'axios'
+import userEvent from "@testing-library/user-event";
+
 import ModalStatus from './modal-status';
+import NoMatchRedirect from '../../pages/noMatchRedirect';
+import { UserContext } from '../../util/user-context';
+
+jest.mock('axios');
 
 describe('Modal Status Component', () => {
-  test('Modal session status renders', () => {
-    const { getByTestId } = render( <ModalStatus visible={true} />);
-      expect(getByTestId('inactivity')).toBeInTheDocument();
-  })
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Modal session status renders and click continue session', async () => {
+    axiosMock.get.mockImplementation(() => Promise.resolve({ status: 200 }));
+
+    const context = {
```

Comment:
Can you pls create a test config file to keep the context and export that to the test. This is being used in several tests, so it will be great if we can extract that out. Similar thing for the error context.. 
For mapping tests we have created a data.config.js under config directory. Probably not the right place to mix test data config with application configs, and I plan to move them to a better place later.

Reply:
I created a user-context-mock under '/assets/mock-data' since it's mocking Context data and not configuration for the app.